This file is part of MXE.
See index.html for further information.

From 09eb90c58226b667e5ad327520c7bc3ec18617b8 Mon Sep 17 00:00:00 2001
From: Boris Nagaev <bnagaev@gmail.com>
Date: Sat, 29 Aug 2015 11:15:52 +0200
Subject: [PATCH] GeoIP.c: include winsock2.h before windows.h

Fix a warning:

In file included from GeoIP.c:35:0:
mxe/usr/i686-w64-mingw32.static/include/winsock2.h:15:2:
warning: #warning Please include winsock2.h before windows.h
[-Wcpp]
 #warning Please include winsock2.h before windows.h
   ^
---
 src/GeoIP.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/GeoIP.c b/src/GeoIP.c
index be2c5e4..e1250e0 100644
--- a/src/GeoIP.c
+++ b/src/GeoIP.c
@@ -31,8 +31,8 @@
 #include <sys/mman.h>
 
 #else
-#include <windows.h>
 #include <winsock2.h>
+#include <windows.h>
 #define snprintf _snprintf
 #endif
 #include <errno.h>
-- 
1.7.10.4

